src/tests/ftest/ior/intercept_messages_pil4dfs.py

```diff
@@ -0,0 +1,49 @@
+"""
+  (C) Copyright 2019-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import re
+import os
+
+from ior_test_base import IorTestBase
+
+
+class IorInterceptMessagesPil4dfs(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with interception.
+
+       Look for messages provided by the library
+
+    :avocado: recursive
+    """
+
+    def test_ior_intercept_messages_pil4dfs(self):
+        """Jira ID: DAOS-12142.
+
+        Test Description:
+            Purpose of this test is to run ior using dfuse with interception
+            library libpil4dfs enabled and look for some debug messages provided
+            by the library to make sure IL is loaded.
+
+        Use case:
+            Run ior with dfuse flags set on debug
+            Look for interception library messages.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=daosio,dfuse,il,ior,ior_intercept,pil4dfs
+        :avocado: tags=IorInterceptMessagesPil4dfs,test_ior_intercept_messages_pil4dfs
+        """
+        intercept = os.path.join(self.prefix, 'lib64', 'libpil4dfs.so')
+        self.ior_cmd.env['D_IL_REPORT'] = "true"
```

Comment:
Curious - how is "true" different from 0, 1, etc.?

Reply:
D_IL_REPORT supports different value (e.g., 0,1,2,-1 etc) in libioil.so. Right now, we only supports a binary value (true / false) associated with D_IL_REPORT in libpil4dfs.so. "false" and "0" as well as non-set  D_IL_REPORT are mapped to false. All other cases are mapped to true. It is just a switch to control whether logs are printed or not.

Reply:
Thanks for the explanation!

Reply:
is it logging or usage statistics?